src/redux/sagas/dBConnectionSaga.tsx

```diff
@@ -0,0 +1,29 @@
+import axios, { AxiosResponse } from "axios";
+import { all, call, put, takeLatest } from "redux-saga/effects";
+import { DB_CONNECTION_REQUEST } from "../actions/actionTypes";
+import { dBConnectionPostAction, dBConnectionPostFailure, dBConnectionPostSuccess } from "../actions/dbConnectionAction";
+
+const dBConnectionPostAPI = 'http://localhost:8080/sourcecourse/db';
+
+function dBConnectionPostAPIcall(name: any, description: any, connectionURL: any, username: any, password: any): Promise<AxiosResponse<any, any>> {
+    console.log("dBConnectionPostAPIcall");
```

Comment:
Removing console is applicable for all files

Reply:
Done

Reply:
Done